extensions/amp-recaptcha-input/0.1/test/test-amp-recaptcha-input.js

```diff
@@ -15,21 +15,71 @@
  */
 
 import '../amp-recaptcha-input';
+import {toggleExperiment} from '../../../../src/experiments';
 
 describes.realWin('amp-recaptcha-input', {
-  win: { /* window spec */
-    location: '...',
-    historyOff: false,
-  },
   amp: { /* amp spec */
     runtimeOn: false,
     extensions: ['amp-recaptcha-input'],
   },
-}, () => {
+}, env => {
+
+  let win;
+  let doc;
+  beforeEach(() => {
+    win = env.win;
+    doc = win.document;
+    toggleExperiment(win, 'amp-recaptcha-input', true);
+  });
+
+  function getRecaptchaInput() {
+    const ampRecaptchaInput = doc.createElement('amp-recaptcha-input');
+    ampRecaptchaInput.setAttribute('layout',
+        'nodisplay');
+    ampRecaptchaInput.setAttribute('data-sitekey',
+        '6LebBGoUAAAAAHbj1oeZMBU_rze_CutlbyzpH8VE');
+    doc.body.appendChild(ampRecaptchaInput);
+    return ampRecaptchaInput.build().then(() => {
+      return ampRecaptchaInput.layoutCallback();
+    }).then(() => {
+      return ampRecaptchaInput;
+    });
+  }
 
   describe('amp-recaptcha-input', () => {
-    it('should pass', () => {
-      expect(true).to.be.ok;
+
+    it('Should be visible after built', () => {
+      return getRecaptchaInput().then(ampRecaptchaInput => {
+        expect(win.getComputedStyle(ampRecaptchaInput).display)
+            .to.not.equal('none');
+      });
+    });
+
+    it('Should apply styles aftyer build', () => {
+      return getRecaptchaInput().then(ampRecaptchaInput => {
+        expect(win.getComputedStyle(ampRecaptchaInput).visibility)
```

Comment:
`position: absolute` sounds like the most important one.  You want to add a check to that?